core/src/test/java/io/aiven/klaw/service/AclControllerServiceTest.java

```diff
@@ -241,8 +241,8 @@ public void createAclConsumerFailure() throws KlawException {
     acl.setConsumergroup(aclRequestsModel.getConsumergroup());
 
     when(commonUtilsService.getTopicsForTopicName(anyString(), anyInt())).thenReturn(topicList);
-    when(handleDbRequests.getUniqueConsumerGroups(anyInt()))
-        .thenReturn(Collections.singletonList(acl));
+    when(handleDbRequests.validateIfConsumerGroupUsedByAnotherTeam(anyInt(), anyInt(), anyString()))
+        .thenReturn(true);
     stubUserInfo();
     mockKafkaFlavor();
```

Comment:
what do you think about adding a new failing test here https://github.com/Aiven-Open/klaw/blob/main/core/src/test/java/io/aiven/klaw/TopicAclControllerIT.java#L671

Reply:
@muralibasani could you please clarify why do we need it?
My assumption was that it is only internal change which should not impact end to end behavior, i.e. all existing tests should become green

Reply:
@snuyanzin we do not have any tests already (previously) to test if a consumer group is owned by another team. So this might be the right time to add it.

Reply:
The test for this case is added here https://github.com/Aiven-Open/klaw/pull/1574/files#diff-a679697f4f9c39c78a6eb235b7660b9710057c557adcf23c5e7bfc479e029d83R808-R813